openai/openai.go

```diff
@@ -218,6 +218,9 @@ func toChatCompletion(id string, r api.ChatResponse) ChatCompletion {
 			Index:   0,
 			Message: Message{Role: r.Message.Role, Content: r.Message.Content, ToolCalls: toolCalls},
 			FinishReason: func(reason string) *string {
+				if len(toolCalls) > 0 {
+					return "tool_calls"
```

Comment:
```suggestion
					reason = "tool_calls"
```

This will not build, you cannot return a string constant if the function expects a `*string`. Instead, set the `reason`. It'll be returned properly in line 225.

Reply:
oh, okay, make sense, thanks